src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -284,8 +284,8 @@
   <data name="FancyZones_KeepWindowsPinned.Content" xml:space="preserve">
     <value>Keep windows pinned to multiple desktops in the same zone when the active desktop changes</value>
   </data>
-  <data name="FancyZones_LaunchEditorButtonControl_Header.Content" xml:space="preserve">
-    <value>Launch Zones Editor</value>
+  <data name="FancyZones_LaunchEditorButtonControl.Text" xml:space="preserve">
+    <value>Launch Zones editor</value>

```

Comment:
Since Zones is not the name of the editor, let's make it lowercase as well.

Reply:
Good catch! Part of the latest commit